lib/pluginmanager/bundler/logstash_injector.rb

```diff
@@ -20,7 +22,6 @@ def self.inject!(new_deps, options = { :gemfile => LogStash::Environment::GEMFIL
       injector.inject(gemfile, lockfile)
     end
 
-
     # This class is pretty similar to what bundler's injector class is doing
```

Comment:
we're doing different things, compared to the original implementation of the `inject` method, so this comment should be updated